libs/patterns.py

```diff
@@ -233,8 +352,8 @@ def _check_rule_is_cidr(self, data_sources, conditions):
             ipaddress.ip_network(data_sources["source"])
         except ValueError:
             return not conditions["is_cidr"]
-        except Exception as e:
-            self._logger.debug(f'Error in creating ip_network from {data_sources["source"]}: {e}')
+        except Exception as err_msg:
```

Comment:
Why are you so mean with my `e` variable name? 😭

Reply:
It's not conform to PEP 8
https://pycodequ.al/docs/pylint-messages/c0103-invalid-name.html
https://www.python.org/dev/peps/pep-0008/#prescriptive-naming-conventions